src/Point/Point.cs

```diff
@@ -101,6 +103,62 @@ public async Task SendSciMessage(SciMessage message)
             await Connection.SendSciMessage(message.Message.ToByteArray());
         }
 
+        /// <summary>
+        /// Send a SCI message with a Protocol Error.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        public async Task SendSciMessagePDIError(Empty request)
+        {
+            if (Connection == null) throw new InvalidOperationException("Connection is null. Did you call Connect()?");
+            // Arbitrary message, that is not protocol type SCI-P.
+            byte[] message = new Messages.Baseline4R1.CCPdiVersionCheckCommand(
+                _config.LocalId,
+                _config.RemoteId,
+                _config.PDIVersion).ToByteArray();
+            _logger.LogInformation("Sending SCI message: {}", message);
+            await Connection.SendSciMessage(message);
+        }
+
+        /// <summary>
+        /// Send a SCI message with content with a Content Telegram Error.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        public async Task SendSciMessageContentError(Empty request)
+        {
+            if (Connection == null) throw new InvalidOperationException("Connection is null. Did you call Connect()?");
+            byte[] messageBytes = new Messages.Baseline4R1.PointPointPositionMessage(

```

Comment:
this doesn't make a difference, but for now we kept this class free of references to a specific baseline. Should we -- just for clarity -- check the configured baseline and then construct either a Baseline4R1 or R2 message? If this message contents should change in future baselines, we would know that we have to update this portion of the code.

Reply:
Good point, though I feel like it's already quite messy and we should change this at some point.